src/bp/ui-shared/src/Commander/style.scss

```diff
@@ -0,0 +1,69 @@
+.categoryWrapper {
+  min-width: 60px;
```

Comment:
this might be a preference, but would you mind putting all the css properties in alphabetical order? I believe you can select the properties, use the command palette in vs code and type alphabetical order to do it or something like that

Reply:
Do you think that's something that could be configured as a default in .vscode/settings.json ? 
![image](https://user-images.githubusercontent.com/42552874/79792329-b52a6980-831c-11ea-9b95-cb7f70ba8fdc.png) 

I will never remember to do that haha

Reply:
haha I never looked into it, don't have time right now, but I'll take sometime to look into it this week, I'm pretty sure it's possible

Reply:
Leaving it as-is, since I don't really feel like sorting manually all those keys haha, we definitely need something to do it